source/localizable/_code-of-conduct.pl.md

```diff
@@ -2,52 +2,52 @@ Cel
 ---
 
 Podstawowym celem wszelkich konferencji i grup użytkowników powołujących się na ten Kodeks postępowania jest otwartość na jak największą liczbę osób o jak najbardziej urozmaiconych i różnorodnych korzeniach. Z tego powodu zobowiązujemy się do zapewnienia życzliwej, bezpiecznej i przyjaznej atmosfery wszystkim, bez względu na płeć, orientację seksualną, sprawność, pochodzenie etniczne, status społeczno-ekonomiczny i religię (lub jej brak).
-
+https://creativecommons.org/licenses/by-sa/4.0/deed.pl
```

Comment:
I think this link might’ve gotten mis-pasted here.

Reply:
It totally was :) I'm blaming my middle mouse button.

Reply:
Removed the link.